src/Batch/FileStaging/Src/AzureBatchFileStaging/AssemblyAttributes.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("Microsoft.Azure.Batch.FileStaging")]
+[assembly: AssemblyDescription("Client library for uploading resource files for tasks in the Azure Batch service.")]
+
+[assembly: AssemblyVersion("5.1.2.0")]
```

Comment:
Why? Can't the file staging assembly track differently than the main assembly? (Like file conventions does)

Obviously the `Microsoft.Azure.Batch` assembly needs to go to 6.0, but the FileStaging assembly doesn't